DEVELOP.rst

```diff
@@ -44,17 +44,17 @@ Distributing
 
 To create a release, you must:
 
- - Update the version in ``src/crate/theme/rtd/__init__.py``
- - Update the version in ``package.json``
- - Add section to the ``CHANGES.txt`` file
- - Commit your changes with a message like "prepare release x.x.x"
- - Push to origin on the master branch
- - Run ``./devtools/create_tag.sh``
- - Run ``bin/py setup.py sdist upload``
+- Update the version in ``src/crate/theme/rtd/__init__.py``
+- Update the version in ``package.json``
+- Add section to the ``CHANGES.txt`` file
+- Commit your changes with a message like "prepare release x.x.x"
+- Push to origin on the master branch
+- Run ``./devtools/create_tag.sh``
+- Run ``bin/py setup.py sdist upload``
```

Comment:
@mfussenegger am I okay to merge as is for the formatting fixes and then do another PR for updating the docs? or should I combine changes?